_data/meltano/loaders/target-clickhouse/shaped.yml

```diff
@@ -0,0 +1,30 @@
+description: Clickhouse Database Loader
+dialect: clickhouse
+domain_url: https://www.clickhouse.com/
+hidden: false
+keywords:
+- database
+label: ClickHouse
+logo_url: /assets/logos/targets/clickhouse.png
+maintenance_status: active
+name: target-clickhouse
+namespace: target_clickhouse
+pip_url: git+https://github.com/shaped-ai/target-clickhouse.git
+quality: silver
+repo: https://github.com/shaped-ai/target-clickhouse.git
+settings:
+- description: |
+    Lets you set `user`, `password`, `host`, `port`, and `dbname` in one
+    go using a `clickhouse://` URI.
+
+    Takes precedence over the other settings when set.
+  kind: string
+  label: SQLAlchemyUrl
+  name: sqlalchemyurl
+settings_group_validation: []
+target_schema: $TARGET_CLICKHOUSE_SCHEMA
+usage: |
+  ## Troubleshooting
+
+  TODO
+variant: shaped-ai
```

Comment:
```suggestion
capabilities:
- about
- schema-flattening
- stream-maps
- target-schema
description: Clickhouse Database
domain_url: https://www.clickhouse.com/
executable: target-clickhouse
keywords:
- meltano_sdk
- database
label: Clickhouse
logo_url: /assets/logos/loaders/clickhouse.png
maintenance_status: active
name: target-clickhouse
namespace: target_clickhouse
next_steps: ''
pip_url: git+https://github.com/shaped-ai/target-clickhouse.git
repo: https://github.com/shaped-ai/target-clickhouse
settings:
- description: The default target database schema name to use for all streams.
  kind: string
  label: Default Target Schema
  name: default_target_schema
- description: "'True' to enable schema flattening and automatically expand nested\
    \ properties."
  kind: boolean
  label: Flattening Enabled
  name: flattening_enabled
- description: The max depth to flatten schemas.
  kind: integer
  label: Flattening Max Depth
  name: flattening_max_depth
- description: SQLAlchemy connection string
  kind: password
  label: SQLAlchemy URL
  name: sqlalchemy_url
- description: User-defined config values to be used within map expressions.
  kind: object
  label: Stream Map Config
  name: stream_map_config
- description: Config object for stream maps capability. For more information check
    out [Stream Maps](https://sdk.meltano.com/en/latest/stream_maps.html).
  kind: object
  label: Stream Maps
  name: stream_maps
settings_group_validation:
- - sqlalchemy_url
settings_preamble: ''
usage: ''
variant: shaped-ai
```

@BTheunissen we have some scripts that we've been using to add plugins to the hub using the plugins `--about` output, soon we'll let the automation do it all for us but its still a work in progress right now. This is the output of the scripts, feel free to edit or accept as it is.